### ISSUE TITLE
Wiki not linked from program

### ISSUE THREAD

**Issue by hoobhab** [fiji/fiji#374]
There isn’t a link to the FIJI wiki from the program. While the ImageJ website is accessible, the wiki does contain useful information about contributing guidelines that isn’t on the website.

**Comment by ctrueden** [comment]
@hoobhab Yeah, because Fiji is bolted on top of the original ImageJ user interface. So the *Help &rarr; ImageJ Website* command opens the ImageJ website. The ImageJ *wiki* was an effort by the ImageJ2 project to create a unified place to document plugins for both ImageJ and ImageJ2, including all functionality available in Fiji or via update sites. But it was never embraced by @rasband, and the content of the wiki remains completely separated from the ImageJ website at https://imagej.net/ij.

What would you suggest? A new menu item *Help &rarr; ImageJ wiki* that opens https://imagej.net ?

**Comment by ctrueden** [comment]
See also [this forum post](https://forum.image.sc/t/connection-problems-with-imagej-nih-gov-ij-developer-macro-functions-html/5047/5?u=ctrueden).
